molgenis-api-identities/src/main/java/org/molgenis/api/identities/IdentitiesApiController.java

```diff
@@ -217,6 +229,89 @@ public void updateMember(
     groupService.updateMemberRole(group, member, newRole);
   }
 
+  @GetMapping(VO_GROUP_MEMBER_END_POINT)
+  @ApiOperation(value = "Get VO group members", response = Collection.class)
+  @ResponseBody
+  public Collection<VOGroupMemberResponse> getVOGroupMembers(
+      @PathVariable(value = "groupName") String groupName) {
+    checkGroupPermission(groupName, GroupPermission.VIEW_MEMBERSHIP);
+    Iterable<Role> roles = groupService.getGroup(groupName).getRoles();
+    return voGroupRoleMembershipService.getMemberships(Lists.newArrayList(roles)).stream()
+        .map(VOGroupMemberResponse::fromEntity)
+        .collect(Collectors.toList());
+  }
+
+  @PostMapping(VO_GROUP_MEMBER_END_POINT)
+  @ApiOperation(value = "Add VO group to group", response = ResponseEntity.class)
+  @Transactional
+  @ApiResponses({
+    @ApiResponse(code = 201, message = "VO group added to group", response = ResponseEntity.class)
+  })
+  public ResponseEntity addVOGroupMember(
+      @PathVariable(value = "groupName") String groupName,
+      @RequestBody AddVOGroupMemberCommand addMemberCommand) {
+    checkGroupPermission(groupName, GroupPermission.ADD_MEMBERSHIP);
+    final Group group = groupService.getGroup(groupName);
+    final String voGroupName = addMemberCommand.getVOGroup();
+    final String roleName = addMemberCommand.getRoleName();
+    final Role role = roleService.getRole(roleName);
+    final VOGroup voGroup = voGroupService.getGroup(voGroupName);
+
+    groupService.addVOGroupMembership(group, voGroup, role);
+
+    URI location =
+        ServletUriComponentsBuilder.fromCurrentRequest()
+            .path("/{voGroupName}")
```

Comment:
Mhmm... now the name ends up in the URL after all. Or use ID?

Reply:
Discussed with @sidohaakma: let's use the ID instead

Reply:
We do it now!